lib/ReactViews/Story/story-panel.scss

```diff
@@ -92,7 +92,13 @@
   padding: $padding * 1.5 $padding;
   color: $dark;
   font-size: 1rem;
-  max-height: 400px;
+  max-height: 100px;
+  @media (min-height: 700px) {
+    max-height: 200px;
+  }
+  @media (min-height: 900px) {
+    max-height: 400px;
+  }
```

Comment:
Makes story panel content div responsive and avoids covering the whole screen on small devices.